draft-ietf-taps-interface.md

```diff
@@ -1624,7 +1624,7 @@ Preconnection -> EstablishmentError<reason?>
 
 ## Connection Groups {#groups}
 
-Connection Groups can be created using the Clone Action:
+Connection Groups represent sets of entagled connections, e.g., streams in a multi-streaming protocol, and can be created using the Clone Action:
```

Comment:
This change seems unrelated to the main change (and we've gone back and forth here); propose removing it.

Reply:
Reading it without this hint felt a little confusing to me after not reading it a long time. I am happy to remove it, but would prefer third opinion here.

Reply:
I agree to remove it. The text below the Clone call explains it all, and it was revised not too long ago in a PR, so this sits with agreement behind it.

A side note, we no longer talk about Connections being "entangled": the term only applies to their Connection Properties.  Also, using the term here before it's explained isn't good... at best, we could use the "streams in a multi-streaming protocol" example, but if you're missing this, it also fits better as an addition to the text below the call.

Reply:
Agree to remove.

Reply:
Removed it